gradio/cli_env_info.py

```diff
@@ -0,0 +1,23 @@
+""" This file is the part of 'gradio/cli.py' for printing the environment info 
```

Comment:
This very cool @varshneydevansh ! Thank you

My comments are mainly around making the output look a bit nicer.

1. The dependencies under `gradio` and `gradio_client` should be indented. 
2. I would add a new line between `Operating System` and `gradio` line
3. I would add a new line between the last gradio dependency and the `gradio_client` section.

Reply:
Do you think these changes are the ones you wanted?

Reply:
Yes this is great thank you @varshneydevansh !